src/sections/Partners/Partners.module.scss

```
@@ -0,0 +1,68 @@
+@import "../../styles/vars.scss";
+
+.section {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  max-width: 640px;
+
+  a {
+    text-decoration: none;
+  }
+}
+
+.partnersIconsWrapper {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap:20px;
```

Please check if you could use Prettier to apply consistent formatting 😉 
```suggestion
  gap: 20px;
```